answerking_app/services/item_service.py

```diff
@@ -1,68 +1,72 @@
-from answerking_app.models.models import Item, Order, OrderLine
+from decimal import Decimal
+
+from django.db.models import QuerySet
+
+from answerking_app.models.models import Item, OrderLine
 from answerking_app.models.validation.serializers import ItemSerializer
 
 
-def get_all():
-    items = Item.objects.all()
+def get_all() -> QuerySet[Item] | list:
+    items: QuerySet[Item] = Item.objects.all()
 
     if not items:
         return []
 
     return items
 
 
-def get_by_id(item_id):
+def get_by_id(item_id: str) -> Item | None:
     try:
-        item_id = int(item_id)
+        int(item_id)
```

Comment:
isn't this more readable as item_id: int = int(item_id) ?  (same with the other cases)

Reply:
Pyright throws an error for attempting to assign something already declared as a str to an int.

Reply:
I won't if you give it a new name

Reply:
yeah, I don't think it needs a new assignation at all, that's there just to typecheck if it's a valid string, I believe the models converts it to an int even if it gets a string.

Reply:
This has been done as decided in our meeting